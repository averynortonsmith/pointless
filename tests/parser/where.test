
x = foo where a = b
x = foo where {a = b}
x = foo where {a = b;}
x = foo where {a = b; b = c}
x = foo where {a = b
 b = c}
x = foo where {a = b;
 b = c}