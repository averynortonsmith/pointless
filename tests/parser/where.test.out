(Program 
 null
 []
 (Def 
  (Name "x")
  (Where 
   (Name "foo")
   (Object 
    (Def 
     (Name "a")
     (Name "b")))))
 (Def 
  (Name "x")
  (Where 
   (Name "foo")
   (Object 
    (Def 
     (Name "a")
     (Name "b")))))
 (Def 
  (Name "x")
  (Where 
   (Name "foo")
   (Object 
    (Def 
     (Name "a")
     (Name "b")))))
 (Def 
  (Name "x")
  (Where 
   (Name "foo")
   (Object 
    (Def 
     (Name "a")
     (Name "b"))
    (Def 
     (Name "b")
     (Name "c")))))
 (Def 
  (Name "x")
  (Where 
   (Name "foo")
   (Object 
    (Def 
     (Name "a")
     (Name "b"))
    (Def 
     (Name "b")
     (Name "c")))))
 (Def 
  (Name "x")
  (Where 
   (Name "foo")
   (Object 
    (Def 
     (Name "a")
     (Name "b"))
    (Def 
     (Name "b")
     (Name "c"))))))
